assembl/static2/js/app/components/administration/survey/save.js

```diff
@@ -66,12 +66,16 @@ const getChildrenVariables = (client, thematic, initialTheme): Array<Promise<mix
           t.messageViewOverride.value === MESSAGE_VIEW.survey
           ? convertRichTextToVariables(t.announcement.quote, client)
           : Promise.resolve({ attachments: [], entries: [] }));
+        const summary = await (t.announcement.summary
+          ? convertRichTextToVariables(t.announcement.summary, client)
+          : Promise.resolve({ attachments: [], entries: [] }));
```

Comment:
Good idea 😮 this is the syntax we should follow in every save file for richtext entries I think